wdl-engine/tests/inputs/yaml-only/error.txt

```diff
@@ -0,0 +1,5 @@
+Failed to parse input file `tests/inputs/yaml-only/inputs.yaml`: failed to parse input file `tests/inputs/yaml-only/inputs.yaml`
```

Comment:
This error message shouldn't be present if all we're doing is testing yaml inputs.

At any rate, I think we can delete both the `yaml-only` and `yaml-support` test cases as they're not adding anything new to the test matrix; if instead we do the above and add `inputs.yaml` to each test case, we'll ensure that the existing tests all work with YAML inputs too.